accepted/00076-configuration-channels-visibility.md

```diff
@@ -0,0 +1,81 @@
+- Feature Name: Enhance visibility of connected Configuration/State Channels
+- Start Date: 2020-12-17
+- RFC PR: 
+
+# Summary
+[summary]: #summary
+
+Provide visibility and overview of **inherited** Configuration and State Channels connections at a System Details level.
+
+# Motivation
+[motivation]: #motivation
+
+### Why are we doing this?
+
+At the moment there is no way to figure out Configuration and State Channels connections other than the ones directly in place to a specific System. Such a type of channel can exist and be difined at:
+
+- Organization level
+- System Group level
+- System level
+
+but only the third level is visible for each System. This means that once Salt States are applied, there will be more than what is visible: Salt States are a set of defined states from
```

Comment:
- There is some repeated information in this section (e.g. the fact that only direct channel-system association is visible).
- If I didn't know the topic up to some point, I'd be a bit confused (for instance "This means that once Salt States are applied, there will be more than what is visible" sentence).

I suggest that we re-phrase it in the following way:

```suggestion
At the moment there is no way to figure out which Configuration/State Channels are assigned to a System, other than the directly assigned ones.

Apart from this direct assignment, the System inherits the Configuration/State channels defined on the Organization level and the System Group level, but this inheritance is not visible in the Web UI/XML RPC.

To sum it up: The Configuration/State channels effective for a system are composed based on:
```

Reply:
Please don't forget about the Uyuni level (global), and the formulas with forms level.

Reply:
> * If I didn't know the topic up to some point, I'd be a bit confused (for instance "This means that once Salt States are applied, there will be more than what is visible" sentence).

I see the point, but the RFC is not meant to explain the feature from scratch, otherwise it would be much longer. And the focus would diverge to something which is not intended to be discussed.

In agreement with your comment and others, I am rephrasing using words from all of you.